packages/rum-core/src/rawRumEvent.types.ts

```diff
@@ -38,6 +38,7 @@ export interface RawRumResourceEvent {
   _dd?: {
     trace_id?: string
     span_id?: string // not available for initial document tracing
+    rule_psr: number
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏to be consistent with schema
```suggestion
    rule_psr?: number
```